modules/ROOT/pages/monitoring/metrics/reference.adoc

```diff
@@ -361,7 +394,7 @@ The total number of queries executed remotely to a member of a different cluster
 |<prefix>.cluster.raft.last_leader_message|The time elapsed since the last message from a leader in milliseconds. Should reset periodically. (gauge)
 |===
 
-.Database secondary Metrics
+.Database secondary Metrics / Store copy metrics
```

Comment:
I assume 'Database secondary metrics' could have been 'Store copy metrics', as
the test results show that 'Metric class `Store copy metrics` not found in docs.' However, there is no information about metric class 'Database secondary metrics'.  It might be an extra metric class.